docs/how-to's/kuadrant-addon-walkthrough.md

```diff
@@ -0,0 +1,65 @@
+# Kuadrant operator addon
+
+## Introduction
+The following walkthrough will show you how to install/setup the Kuadrant operator via OCM (Open cluster management addons)
+
+## Prerequisites
+* Kind
+* OLM 
+
+## Open terminal sessions
+For this walkthrough, we're going to use multiple terminal sessions/windows, all using `multicluster-gateway-controller` as the `pwd`.
+
+Open 2 windows, which we'll refer to throughout this walkthrough as:
+
+* `T1` (Hub/control plane cluster, Where we'll run our controller locally)
+* `T2` (Spoke/workload cluster 1)
```

Comment:
I liked the use of T1 and T2 in the ocm walkthough.
Could we make it more obvious with each cmd below which terminal it should be run in similar to the ocm walkthrough?

Reply:
Sure ill add that